### ISSUE TITLE
Transfer Learning Integration

### ISSUE THREAD

**Issue by Tanyajain2006** [Tanyajain2006/Waste-Segregation#1]
Replace the custom CNN with a pre-trained architecture like ResNet50, MobileNetV2, or EfficientNet.
Fine-tune the network to achieve higher accuracy and faster convergence on smaller datasets.

**Comment by ATOMworkplace** [comment]
@Tanyajain2006 Hey, I’d like to work on this issue. Could you please assign it to me?

**Comment by Tanyajain2006** [comment]
Hey @ATOMworkplace, what's the status of work?

**Comment by ATOMworkplace** [comment]
> Hey [@ATOMworkplace](https://github.com/ATOMworkplace), what's the status of work?

Yes working on it I'll push a pr soon
